### ISSUE TITLE
Show news before attempting upgrade

### ISSUE THREAD

**Issue by kyak** [actionless/pikaur#726]
```
pikaur -Vq

```
Pikaur v1.15.3
Pacman v6.0.2 - libalpm v13.0.2 - pyalpm v0.10.6

##### Description:

Arch Linux site news need to be shown before attempting to upgrade.

At the moment, upgrade fails (e.g. with this error https://archlinux.org/news/openblas-0323-2-update-requires-manual-intervention/) and there is no indication from pikaur that site news are available.

The whole point of showing news in pikaur is showing them *before* upgrade, not after.

**Comment by actionless** [comment]
it probably was already shown during the previous upgrade and you didn't noticed that, you can verify that by checking the content of the  `~/.cache/pikaur/last_seen_news.dat`

**Comment by kyak** [comment]
@actionless 

The sequence of events was as following.

1. I was already aware of Arch Linux news from other sources. The news was already posted.
2. I ran recommended manual intervention steps replacing pacman with pikaur.
3. I ran into issue discussed in #727 
4. I never saw any site news from pikaur at this moment, although it had been posted.
5. After I got the issue resolved (by using pacman instead of pikaur) I ran pikaur again.
6. At this time, pikaur has shown me the site news.

This really looked like pikaur is trying to update first, and checks site news later. If update fails, user will not see site news thus defeating the purpose of showing site news in pikaur.
